Presentation/Presentable.swift

```diff
@@ -53,3 +53,11 @@ public protocol Presentable {
     /// Constructs a matter from `self` and returns it toghether with the result of presenting it.
     func materialize() -> (Matter, Result)
 }
+
+// MARK: - Convinience overloads
+public extension Presentable where Result == Void {
+    func materialize() -> Matter {
+        let result: (Matter, Result) = self.materialize()
```

Comment:
Isn't this recursive?

Reply:
No, since I'm specifying the result type the compiler knows what overload to use so it will pick up `func materialize() -> (Matter, Result)` and then I'm returning only the matter. 

A different question is, should it be implemented like this (as an overload), or with a different name, say `func materializeSkippingResult() -> Matter` or something. The downside of having the overload is that if you have type A that conforms to Presentable and its Result is Void, then you need to type
`let view: UIView = A().materialize()`
instead of
`let view = A().materialize()`